.azure-pipelines/linux-CI-nightly.yml

```diff
@@ -10,7 +10,7 @@ jobs:
 
 - job: 'Test'
   pool:
-    vmImage: 'Ubuntu-16.04'
+    vmImage: 'ubuntu-latest'
```

Comment:
can you separate these CI pipeline changes into another PR?

Reply:
Done for all CI files.

Reply:
But the PR fails now so either I fix the examples which fails for ubuntu 16.04 or I write on the example it works for ubuntu 18.04.

Reply:
It looks the failure happening on the more recent ONNXRuntime build. was it caused by the new ORT not supporting ubuntu 16.04 any more?

Reply:
I'm trying to make it work for that distribution.

Reply:
Fixed.